spec/requests/api/v2/profiles_controller_create_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::ProfilesController do
+  let(:response_json) { JSON.parse(response.body) }
+  let(:access_token) { create(:access_token).token }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+  let(:person) { create(:person_without_profiles) }
+  let(:risk_type_1) { create :assessment_question, :risk }
+  let(:risk_type_2) { create :assessment_question, :risk }
+
+  describe 'POST /v2/people/:id/profiles' do
+    let(:schema) { load_yaml_schema('post_profile_responses.yaml', version: 'v2') }
```

Comment:
this spec is currently failing as require pending PR for V2 swagger to be merged, after that should be ok

Reply:
ok, we paired on this and we found a solution

Reply:
🤞 😄